demo/index.html

```diff
@@ -1,21 +1,42 @@
 <!DOCTYPE html>
-<html >
+<html lang="en">
 <head>
+  <title>Minimal Dev Wallet</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <!--Import Google Icon Font-->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+  <!-- Compiled and minified Materialize JS/CSS -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+
+  <!--Credential Handler API Polyfill-->
   <script src="https://unpkg.com/credential-handler-polyfill@2.0.0/dist/credential-handler-polyfill.min.js"></script>
```

Comment:
Update version

Reply:
Since this is just a demo, might want to use a less exact form, but leave a comment that production should probably use properly cached version.  See https://unpkg.com/ for other forms.  Can do latest or semver like versions.